yarn-project/pxe/src/database/memory_db.ts

```diff
@@ -44,11 +45,20 @@ export class MemoryDB extends MemoryContractDatabase implements Database {
     return Promise.resolve(this.authWitnesses[messageHash.toString()]);
   }
 
-  public addNote(note: NoteDao) {
+  public addNote(note: NoteDao): Promise<void> {
     this.notesTable.push(note);
     return Promise.resolve();
   }
 
+  public addMint(mint: Fr[]): Promise<void> {
```

Comment:
Maybe pushSlow?

Reply:
I'd go for `pushToSlowUpdatesTree()` to make it easy for external devs